app/server.js

```diff
@@ -1,10 +1,16 @@
 const app = require('express')();
 const nconf = require('nconf');
 const globSync = require('glob').sync;
+const morgan = require('morgan');
 
 const { errorHandler, notFound } = require('./middleware/error_handler');
 
 const initApp = () => {
+  const env = process.env.NODE_ENV || 'dev';
+  if (env === 'dev') {
+    app.use(morgan('dev'));
```

Comment:
replace 'dev' with corresponding parameter

Reply:
`dev` it's one of middleware type. See [here](https://github.com/expressjs/morgan#dev)

Reply:
app.use(morgan(env)) - is not correct?

Reply:
Morgan doesn't have `prod` or `test` types

Reply:
well, I think you see only this line... bad...
if (env === 'dev') {
   app.use(morgan(env));
}
answer the question: how prod or test type will pass in if block?

Reply:
I think that we should use middleware log only in dev. Isn't it?

Reply:
yep

Reply:
That's why I check environment only for dev

Reply:
please change code from `app.use(morgan('dev'));` to `app.use(morgan(env));`

Reply:
[Done](https://github.com/inst-event-emitter/eventemitter-backend/pull/8/commits/b997d989f2f08098df989c881b1efaf7cf89dd23#diff-f53e59f241bb62f0261e245350683e07R11)